### DISCUSSION TITLE
Typography tokens

### DISCUSSION THREAD

**Discussion by JAN-CHOU** [six7/figma-tokens#595] [Q&A] [answered]
when I deleted the typography tokens in the figma tokens plugin, after clicking update, why is it still there in the figma text styles panel, why is it not updated?

**Comment by six7** [comment, marked as answer]
Currently tokens only have a connecting to styles that have a matching name. Meaning, once a token is deleted, it won't delete the style for you.

We'll be addressing this soon, until then you'd have to manually delete.
